test/e2e/parallel/retarget_test.go

```diff
@@ -0,0 +1,51 @@
+package parallel
+
+import (
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/space"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRetargetUserByChangingSpaceTargetCluster(t *testing.T) {
+	// given
+	t.Parallel()
+
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	member1Await := awaitilities.Member1()
+	member2Await := awaitilities.Member2()
+
+	userSignup, mur := NewSignupRequest(awaitilities).
```

Comment:
What do you think about also sharing this space with another user? So the test would cover more complicated case with a space with spacebindgings?

Reply:
good idea - added another test case ffac8bc0

Reply:
:man_facepalming:  53353e3d

Reply:
> 🤦‍♂️ https://github.com/codeready-toolchain/toolchain-e2e/commit/53353e3d9d02520cf5ad0120bf3be0205776056b

sometimes I push similar commits 😄 , maybe I'll try to create git pre-commit hook for it.